### PR TITLE
minor UI fixes on debug page

### PR DESCRIPTION
**Pull request by technophile-04**
### Description 
Intended to solve #218

### CODE REVIEW

File: packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ export const WriteOnlyFunctionForm = ({
   });
 
   return (
-    <div className="flex flex-col gap-3">
+    <div className={`flex gap-3 py-4 ${inputs.length === 0 ? "flex-row justify-between" : "flex-col"}`}>
```
Comment (sverps) [inline packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx:90]: Should probably also check `functionFragment.payable`, cause now a payable function without inputs looks like this:
image

**Review by sverps — CHANGES_REQUESTED** [review]

**Comment by sverps** [comment]
Should we horizontally align the function name with the button? Now it looks like the function name is slightly offcenter:
image

File: packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx (hunk 1 above)
Comment (technophile-04) [reply packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx]: Tysm fixed it [a4a9913](https://github.com/scaffold-eth/se-2/pull/223/commits/a4a9913c056407bb971457927a58643819d8fe57)

**Review by sverps — APPROVED** [review]

**Review by rin-st — APPROVED** [review]

**Comment by technophile-04** [comment]
- [X] Remove dropdown functionality (besides removing the caret, remove the un-needed classes) -> [2f77fe7](https://github.com/scaffold-eth/se-2/pull/223/commits/2f77fe78c74ef6654adc5a447ca0d34c3e92a452)
- [X] Separator between methods (use base-300 color) -> [d92b072](https://github.com/scaffold-eth/se-2/pull/223/commits/d92b0721c2986327965babe1338e294954ad2d18) , [5e56331](https://github.com/scaffold-eth/se-2/pull/223/commits/5e5633167baa4f9731934e1be209d54c38397328)
- [X] When there is no argument, align the button with the method name. [1c3fe84](https://github.com/scaffold-eth/se-2/pull/223/commits/1c3fe847d57c58bb1443e79e396522e982846654), [d92b072](https://github.com/scaffold-eth/se-2/pull/223/commits/d92b0721c2986327965babe1338e294954ad2d18)
- [X] The asterisk * is too big, let's use the smaller unit [2ebb2d7](https://github.com/scaffold-eth/se-2/pull/223/commits/2ebb2d738cc1bc5951aaba505878c4e536058bb0)
- [X] Top / bottom spacing (between methods and also between the methods and the wrapper) [5e56331](https://github.com/scaffold-eth/se-2/pull/223/commits/5e5633167baa4f9731934e1be209d54c38397328)

Fixes #218

**Review by sverps — CHANGES_REQUESTED** [review]
Don't think we need to pass down the index, we should be able to use css selectors.

File: packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx [hunk 2]
```diff
@@ -87,7 +89,11 @@ export const WriteOnlyFunctionForm = ({
   });
 
   return (
-    <div className="flex flex-col gap-3">
+    <div
+      className={`flex gap-3 ${
+        inputs.length === 0 && !functionFragment.payable ? "flex-row justify-between items-center" : "flex-col"
+      } ${index === 0 ? "pt-1 pb-5" : "py-5"}`}
```
Comment (sverps) [inline packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx:95]: Same here, we can use the `first:` css selector.

File: packages/nextjs/components/scaffold-eth/Contract/ReadOnlyFunctionForm.tsx [hunk 3]
```diff
@@ -55,7 +56,7 @@ export const ReadOnlyFunctionForm = ({ functionFragment, contractAddress }: TRea
   });
 
   return (
-    <div className="flex flex-col gap-3">
+    <div className={`flex flex-col gap-3 ${index === 0 ? "pt-1 pb-5" : "py-5"}`}>
```
[the remark this answers is not in the document]
Comment (rin-st) [reply packages/nextjs/components/scaffold-eth/Contract/ReadOnlyFunctionForm.tsx:59]: or just first for `flex flex-col gap-3 py-5 first:pt-1`

[the remark this answers is not in the document]
Comment (sverps) [reply packages/nextjs/components/scaffold-eth/Contract/ReadOnlyFunctionForm.tsx:59]: ah yes, even better :+1:

File: packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx (hunk 2 above)
Comment (technophile-04) [reply packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx]: 🙌 fixed at [6c57dcd](https://github.com/scaffold-eth/se-2/pull/223/commits/6c57dcdb817ebddefdee227a4309d5298d517590)

**Review by rin-st — APPROVED** [review]

**Review by sverps — APPROVED** [review]

**Comment by technophile-04** [comment]
Just noticed a bug : 
Screenshot 2023-03-02 at 8 58 39 PM

will be fixing it soon

**Review by sverps — APPROVED** [review]

**Comment by carletex** [comment]
This looks amazing @technophile-04 !

And thanks for the reviews y'all!

**Pull request merged**